tests/integration/ardrive.int.test.ts

```diff
@@ -25,18 +25,20 @@ import { expectAsyncErrorThrow } from '../../src/utils/test_helpers';
 import { ArDriveCommunityOracle } from '../../src/community/ardrive_community_oracle';
 import { ArFSDAO, PrivateDriveKeyData } from '../../src/arfsdao';
 import { deriveDriveKey, DrivePrivacy } from 'ardrive-core-js';
-import { DriveKey, FileID, Winston } from '../../src/types';
+import { DriveKey, FileID, W, Winston, FeeMultiple, EID } from '../../src/types';
 import { ArFSFileToUpload, wrapFileOrFolder } from '../../src/arfs_file_wrapper';
+import { RootFolderID } from '../../src/utils/arfs_builders/arfs_folder_builders';
 
-const entityIdRegex = /^([a-f]|[0-9]){8}-([a-f]|[0-9]){4}-([a-f]|[0-9]){4}-([a-f]|[0-9]){4}-([a-f]|[0-9]){12}$/;
+// Don't use the existing constants just to make sure our expectations don't change
+const entityIdRegex = /^[a-f\d]{8}-([a-f\d]{4}-){3}[a-f\d]{12}$/i;
 const trxIdRegex = /^([a-zA-Z]|[0-9]|-|_){43}$/;
 const fileKeyRegex = /^([a-zA-Z]|[0-9]|-|_|\/|\+){43}$/;
```

Comment:
```ts
// Don't use the existing constants just to make sure our expectations don't change
```
But shouldn't these be an exact copy of the original ones?

Reply:
Yes these have migrated apart. We should unify, but it's not critical for this branch.

Reply:
3cb7221